pytorch_lightning/trainer/__init__.py

```diff
@@ -138,6 +138,7 @@ def forward(self, x):
 accumulate_grad_batches
 ^^^^^^^^^^^^^^^^^^^^^^^
 Accumulates grads every k batches or as set up in the dict.
+This argument also calls ``optimizer.step()`` for the last indivisible batch.
```

Comment:
Could be reformulated. It's not the argument that does that, it's the trainer.

Reply:
Okay